.github/workflows/ci-synthetic-test-trigger.yml

```
@@ -0,0 +1,71 @@
+            core.setOutput('target_url', targetUrl);
+            core.setOutput('pr_number', prNumber);
+            core.setOutput('pr_ref', pr.head.ref);
+            core.setOutput('pr_sha', pr.head.sha);
+
+      - name: Cancel in-progress runs for this PR
+        if: steps.check.outputs.should_run != 'true'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo "Cancelling in-progress synthetic test runs for PR #${{ steps.check.outputs.pr_number }}"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)